src/OAuth/Nonce/UniqIdNonceGenerator.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Penneo\SDK\OAuth\Nonce;
+
+class UniqIdNonceGenerator implements NonceGenerator
+{
+    public function generate(): string
+    {
+        return substr(hash('sha512', uniqid('', true)), 0, 64);
```

Comment:
Why not just use `random_bytes(...)`? 

```suggestion
        return \random_bytes(64);
```

Alternative we could just use `SHA256` which has the hex length of 64 characters however it is still just hex lowering the strength of the nonce.

Reply:
No good reason. The original implementation is something I copied over from an example. I've replaced it with `random_bytes`.